frontend/src/components/CANs/CANActivePeriodComboBox/CANActivePeriodComboBox.jsx

```diff
@@ -0,0 +1,58 @@
+import ComboBox from "../../UI/Form/ComboBox";
+/**
+ * @typedef {Object} DataProps
+ * @property {number} id - The identifier of the data item
+ * @property {string | number} title - The title of the data item
+ */
+
+/**
+ * @component
+ * @param {Object} props - The component props.
+ * @param {DataProps} props.activePeriod - The current active period.
+ * @param {Function} props.setActivePeriod - A function to call to set the active period.
+ * @param {string} [props.legendClassname] - The class name for the legend (optional).
+ * @param {string} [props.defaultString] - The default string to display (optional).
+ * @param {Object} [props.overrideStyles] - The CSS styles to override the default (optional).
+ * @returns {JSX.Element} - The rendered component.
+ */
+const CANActivePeriodComboBox = ({
+    activePeriod,
+    setActivePeriod,
+    legendClassname = "usa-label margin-top-0",
+    defaultString = "",
+    overrideStyles = {}
+}) => {
+    const periods = [
+        { id: 1, title: "1 Year" },
+        { id: 2, title: "2 Year" },
+        { id: 3, title: "3 Year" },
+        { id: 4, title: "4 Year" },
+        { id: 5, title: "5 Year" }
+    ];
+
+    return (
+        <div className="display-flex flex-justify">
+            <div>
+                <label
+                    className={legendClassname}
+                    htmlFor="can-active-period-combobox-input"
+                >
+                    Fiscal Year
```

Comment:
@fpigeonjr according to figma design in the For Devs page, this label is supposed to say "Active Period"

<img width="348" alt="Screenshot 2024-10-08 at 02 12 49" src="https://github.com/user-attachments/assets/987f194b-e5df-4285-b2ff-8cd5ad974fae">

Reply:
thanks for pointing this out